packages/cli/src/commands/record/record.command.ts

```diff
@@ -188,5 +191,10 @@ export const recordCommand = buildCommand("record")
       boolean: true,
       description: "Enable verbose logging",
     },
+    captureHttpOnlyCookies: {
+      boolean: true,
+      default: true,
```

Comment:
I wonder if the implicit nature of this divergence between the CLI and the snippet might lead to confusion down the line for cases where simulations work on session recorded via the CLI during onboarding / testing but not for passively recorded ones.

Reply:
Yeah I was worried about that... but at least if it works with CLI sessions but not with normal ones we'd be able to guess why. Chrome extensions [can access httpOnly cookies](https://developer.chrome.com/docs/extensions/reference/cookies/), if we have robot auto-genned tests they will be able to do so too, so my thinking here was that maybe we default this to true, instead of defaulting to the lowest common denominator?